src/services/__tests__/boegivar.test.js

```diff
@@ -0,0 +1,11 @@
+import expect from 'expect';
+import { boegivar } from '../boegivar';
+
+describe('boegivar', () => {
+  describe('sayBoegivar', () => {
+    it('returns a message containing parameter firstName', () => {
```

Comment:
Test null/nil/nan/undefined/whatever cases as well. We don't want the server crashing.